api/organisations/tasks.py

```diff
@@ -30,5 +32,12 @@ def send_org_over_limit_alert(organisation_id):
 
 
 @register_task_handler()
-def update_organisation_subscription_information_caches():
-    subscription_info_cache.update_caches()
+def update_organisation_subscription_information_influx_caches():
+    influx_cache = (SubscriptionCacheEntity.INFLUX,)
```

Comment:
Do we need to separate this out? Can't we just do: 

```python
subscription_info_cache.update_caches((SubscriptionCacheEntity.INFLUX,))
```

If we want to call it out to it's own line, we should update the variable name as it's not `influx_cache`. I'd say we should just reuse the name of the argument in the function? 

```python
update_cache_entities=(SubscriptionCacheEntity.INFLUX,)
```

Same comment applies to the function below as well.

Reply:
Done.